phpseclib/Crypt/Base.php

```diff
@@ -44,9 +44,9 @@
  * @package Base
  * @author  Jim Wigginton <terrafrost@php.net>
  * @author  Hans-Juergen Petrich <petrich@tronic-media.com>
- * @access  public
+ * @abstract
```

Comment:
tbh, i think we should get rid of the package docblocks too. totally useless as far as i can see